bin/bwa_mem.pl

```diff
@@ -365,7 +365,12 @@ =head2 OPTIONAL parameters
 WARNING:
 bwa_mem.pl will exclude all QCFAIL reads from mapping. If a BAM/CRAM file has been created using
 this option please ensure that you pre-process the file to remove the flag 512 if you intend to
-reprocess based on that output.
+reprocess based on that output. 
+
+The script mmFlagModifier -r (--remove) can process a bam file to remove any occurences of 
```

Comment:
Isn't it:

`mmFlagModifier -m (--remove)`

Reply:
Yes, yes it is.